hexa/datasets/tests/test_generate_sample.py

```diff
@@ -200,6 +201,50 @@ def test_generate_sample(
                             sample_entry.status_reason, expected_status_reason
                         )
 
+    @override_settings(WORKSPACE_DATASETS_FILE_SNAPSHOT_SIZE=1)
+    def test_generate_sample_wkb(
+        self,
+    ):
+        CASES = [
+            (
+                "example_with_wkb_geometry.parquet",
+                DatasetFileSample.STATUS_FINISHED,
+                [wkb_geometry],
+                None,
+            ),
+        ]
+        for (
+            fixture_name,
+            expected_status,
+            expected_sample,
+            expected_status_reason,
+        ) in CASES:
+            with self.subTest(fixture_name=fixture_name):
+                fixture_file_path = os.path.join(
+                    os.path.dirname(__file__), f"./fixtures/{fixture_name}"
```

Comment:
[Minor] You are using a mix of Path and string interpolation for the path creation, we might want to stick with Path